build.psm1

```diff
@@ -1320,20 +1325,26 @@ function Publish-NuGetFeed
 {
     param(
         [string]$OutputPath = "$PSScriptRoot/nuget-artifacts",
-        [Parameter(Mandatory=$true)]
-        [string]$VersionSuffix
+        [Parameter(ParameterSetName = "ReleaseTag")]
```

Comment:
I added this because we could return `[string]$VersionSuffix` to _temporary_ fix daily builds.